ui/js/directives/perf/compare.js

```diff
@@ -21,25 +20,33 @@ treeherder.directive(
                     return !matchText || item.toLowerCase().indexOf(matchText.toLowerCase()) > (-1);
                 }
                 scope.filterTest = function(item) {
-                    return filter(item, scope.testFilter);
+                    return _.any(scope.filter.split(' '), function(matchText) {
+                        return filter(item, matchText);
+                    });
```

Comment:
I think we want to make sure that *all* the terms match against the testname (well, test + platform, see below)